pkg/goldpinger/k8s.go

```diff
@@ -16,14 +16,24 @@ package goldpinger
 
 import (
 	"log"
-
+	"io/ioutil"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func GetAllPods() map[string]string {
+func getNamespace() string {
+	b, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```

Comment:
@blakebarnett @seeker89 sorry for jumping on a merged PR like this ;) but we've hit this on one of our clusters and wanted to share.

Although this is a rare case, a cluster may have Service Account admission controller disabled (I know... I know...) in this case this file doesn't exist.

Could we consider a fallback option to read this from an Env variable? we could pass it on using `metadata.namespace` downward api, similar to how we pass down the hostname.